src/features/accessibility/components/AccessibilityFiltersModal.tsx

```diff
@@ -91,38 +110,55 @@ export const AccessibilityFiltersModal: React.FC<AccessibilityModalProps> = ({
         <Spacer.Column numberOfSpaces={8} />
         <StyledCheckBox>
           <Checkbox
-            isChecked={isVisualHandicapCompliant}
+            isChecked={!!disabilitiesProperties?.isVisualDisabilityCompliant}
```

Comment:
je suis chaud que l'on en discute mais j'arrive pas à comprendre pourquoi les disabilities local peuvent être undefined

Reply:
Yes, j'avais noté de te parler de cela, on en discutait hier aussi